nextjs-13/src/components/ListNewsletter/index.tsx

```diff
@@ -1,24 +1,27 @@
+import NewsletterItem from '@components/NewsletterItem';
+
 interface Record {
   id: string;
   name: string;
 }
 
 interface Props {
   records: Record[];
+  onAfterCloseCallback: () => undefined;
 }
 
-const ListNewsletter = ({ records }: Props) => {
+const ListNewsletter = ({ records, onAfterCloseCallback }: Props) => {
   return (
     <div className="list-newsletter" data-testid="list-newsletter">
       <div>
         <ul className="newsletter-wrapper">
-          {records.map((item) => {
-            return (
-              <li className="newsletter-item" key={item.id}>
-                <span>{item.name}</span>
-              </li>
-            );
-          })}
+          {records.map((item) => (
+            <NewsletterItem
+              key={item.id}
+              item={item}
+              onAfterCloseCallback={onAfterCloseCallback}
```

Comment:
Ditto here on the naming:

```suggestion
              onAfterDeleteCallback={refreshRecordListCallback}
```

Reply:
Same as the above issue, resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/88c0be267f1152993e680d082a339f3909afd2d2)